domain/article.go

```diff
@@ -21,7 +21,7 @@ type ArticleUsecase interface {
 	GetByID(ctx context.Context, id int64) (Article, error)
 	Update(ctx context.Context, ar *Article) error
 	GetByTitle(ctx context.Context, title string) (Article, error)
-	Store(context.Context, *Article) error
+	Store(context.Context, art *Article) error
```

Comment:
This will not compile, because you must specify the argument name for both arguments, or not at all.

Also, I think that it makes no sense to specify the name of the argument, since it is already clear from the type what is being passed in this argument.